src/mlpack/CMakeLists.txt

```diff
@@ -88,6 +88,13 @@ foreach(incl_file ${INCLUDE_FILES})
       BYPRODUCTS ${CMAKE_BINARY_DIR}/include/mlpack/${incl_file})
 endforeach()
 
+add_custom_command(TARGET mlpack_headers POST_BUILD
+  COMMENT "HI"
+  COMMAND ${CMAKE_COMMAND} ARGS -E
+  copy ${CMAKE_CURRENT_SOURCE_DIR}/core/data/stopwords.txt
+       ${CMAKE_BINARY_DIR}/include/mlpack/core/data/
+)
```

Comment:
Above addition to copy the stopword.txt file at require place !. Will remove the comment it is just for debugging puprose.

Reply:
I think the user should pass the stop words to CLI. I suggest to remove the file from the repo at all.

Reply:
@lozhnikov , @zoq , both of you suggested to remove the file and hence i removed it.